diff -r -U 3 minecraft\net\minecraft\src\RenderGlobal.java minecraft_patched\net\minecraft\src\RenderGlobal.java
--- minecraft\net\minecraft\src\RenderGlobal.java	Thu Aug 02 00:57:22 2012
+++ minecraft_patched\net\minecraft\src\RenderGlobal.java	Thu Aug 02 00:56:10 2012
@@ -966,7 +966,7 @@
             f2 = f6;
             f3 = f7;
         }
-        float f5 = 0.0004882812F;
+        float f5 = 0.0004882813F;
         double d = (float)field_72773_u + p_72718_1_;
         double d1 = field_72777_q.field_71451_h.field_70169_q + (field_72777_q.field_71451_h.field_70165_t - field_72777_q.field_71451_h.field_70169_q) * (double)p_72718_1_ + d * 0.029999999329447746D;
         double d2 = field_72777_q.field_71451_h.field_70166_s + (field_72777_q.field_71451_h.field_70161_v - field_72777_q.field_71451_h.field_70166_s) * (double)p_72718_1_;
