diff -r -U 3 minecraft_server\net\minecraft\src\Chunk.java minecraft_server_patched\net\minecraft\src\Chunk.java
--- minecraft_server\net\minecraft\src\Chunk.java	Sun Sep 18 14:38:26 2011
+++ minecraft_server_patched\net\minecraft\src\Chunk.java	Sun Sep 18 15:00:05 2011
@@ -72,20 +72,20 @@
    public Chunk(World var1, byte[] var2, int var3, int var4) {
       this(var1, var3, var4);
       this.field_693_b = var2;
-      NibbleArray var10001 = new NibbleArray;
+      //NibbleArray var10001 = new NibbleArray;
       int var10003 = var2.length;
       var1.getClass();
-      var10001.<init>(var10003, 7);
+      NibbleArray var10001 = new NibbleArray(var10003, 7);
       this.field_690_e = var10001;
-      var10001 = new NibbleArray;
+      //var10001 = new NibbleArray;
       var10003 = var2.length;
       var1.getClass();
-      var10001.<init>(var10003, 7);
+      var10001 = new NibbleArray(var10003, 7);
       this.field_689_f = var10001;
-      var10001 = new NibbleArray;
+      //var10001 = new NibbleArray;
       var10003 = var2.length;
       var1.getClass();
-      var10001.<init>(var10003, 7);
+      var10001 = new NibbleArray(var10003, 7);
       this.field_688_g = var10001;
    }
 
diff -r -U 3 minecraft_server\net\minecraft\src\ChunkLoader.java minecraft_server_patched\net\minecraft\src\ChunkLoader.java
--- minecraft_server\net\minecraft\src\ChunkLoader.java	Sun Sep 18 14:38:32 2011
+++ minecraft_server_patched\net\minecraft\src\ChunkLoader.java	Sun Sep 18 15:00:06 2011
@@ -166,47 +166,47 @@
       int var3 = var1.func_395_d("zPos");
       Chunk var4 = new Chunk(var0, var2, var3);
       var4.field_693_b = var1.func_400_i("Blocks");
-      NibbleArray var10001 = new NibbleArray;
+      //NibbleArray var10001 = new NibbleArray;
       byte[] var10003 = var1.func_400_i("Data");
       var0.getClass();
-      var10001.<init>(var10003, 7);
+      NibbleArray var10001 = new NibbleArray(var10003, 7);
       var4.field_690_e = var10001;
-      var10001 = new NibbleArray;
+      //var10001 = new NibbleArray;
       var10003 = var1.func_400_i("SkyLight");
       var0.getClass();
-      var10001.<init>(var10003, 7);
+      var10001 = new NibbleArray(var10003, 7);
       var4.field_689_f = var10001;
-      var10001 = new NibbleArray;
+      //var10001 = new NibbleArray;
       var10003 = var1.func_400_i("BlockLight");
       var0.getClass();
-      var10001.<init>(var10003, 7);
+      var10001 = new NibbleArray(var10003, 7);
       var4.field_688_g = var10001;
       var4.field_687_h = var1.func_400_i("HeightMap");
       var4.field_681_n = var1.func_402_l("TerrainPopulated");
       int var10;
       if(!var4.field_690_e.func_560_a()) {
-         var10001 = new NibbleArray;
+         //var10001 = new NibbleArray;
          var10 = var4.field_693_b.length;
          var0.getClass();
-         var10001.<init>(var10, 7);
+         var10001 = new NibbleArray(var10, 7);
          var4.field_690_e = var10001;
       }
 
       if(var4.field_687_h == null || !var4.field_689_f.func_560_a()) {
          var4.field_687_h = new byte[256];
-         var10001 = new NibbleArray;
+         //var10001 = new NibbleArray;
          var10 = var4.field_693_b.length;
          var0.getClass();
-         var10001.<init>(var10, 7);
+         var10001 = new NibbleArray(var10, 7);
          var4.field_689_f = var10001;
          var4.func_353_b();
       }
 
       if(!var4.field_688_g.func_560_a()) {
-         var10001 = new NibbleArray;
+         //var10001 = new NibbleArray;
          var10 = var4.field_693_b.length;
          var0.getClass();
-         var10001.<init>(var10, 7);
+         var10001 = new NibbleArray(var10, 7);
          var4.field_688_g = var10001;
          var4.func_348_a();
       }
diff -r -U 3 minecraft_server\net\minecraft\src\ChunkProvider.java minecraft_server_patched\net\minecraft\src\ChunkProvider.java
--- minecraft_server\net\minecraft\src\ChunkProvider.java	Sun Sep 18 14:38:28 2011
+++ minecraft_server_patched\net\minecraft\src\ChunkProvider.java	Sun Sep 18 15:01:28 2011
@@ -29,9 +29,9 @@
 
 
    public ChunkProvider(World var1, IChunkLoader var2, IChunkProvider var3) {
-      EmptyChunk var10001 = new EmptyChunk;
+      //EmptyChunk var10001 = new EmptyChunk;
       var1.getClass();
-      var10001.<init>(var1, new byte[256 * 128], 0, 0);
+      EmptyChunk var10001 = new EmptyChunk(var1, new byte[256 * 128], 0, 0);
       this.field_28061_b = var10001;
       this.field_28063_g = var1;
       this.field_28066_d = var2;
diff -r -U 3 minecraft_server\net\minecraft\src\ChunkProviderServer.java minecraft_server_patched\net\minecraft\src\ChunkProviderServer.java
--- minecraft_server\net\minecraft\src\ChunkProviderServer.java	Sun Sep 18 14:38:26 2011
+++ minecraft_server_patched\net\minecraft\src\ChunkProviderServer.java	Sun Sep 18 15:01:27 2011
@@ -28,9 +28,9 @@
 
 
    public ChunkProviderServer(WorldServer var1, IChunkLoader var2, IChunkProvider var3) {
-      EmptyChunk var10001 = new EmptyChunk;
+      //EmptyChunk var10001 = new EmptyChunk;
       var1.getClass();
-      var10001.<init>(var1, new byte[256 * 128], 0, 0);
+      EmptyChunk var10001 = new EmptyChunk(var1, new byte[256 * 128], 0, 0);
       this.field_724_b = var10001;
       this.field_726_g = var1;
       this.field_729_d = var2;
diff -r -U 3 minecraft_server\net\minecraft\src\EntityPlayerMP.java minecraft_server_patched\net\minecraft\src\EntityPlayerMP.java
--- minecraft_server\net\minecraft\src\EntityPlayerMP.java	Sun Sep 18 14:38:22 2011
+++ minecraft_server_patched\net\minecraft\src\EntityPlayerMP.java	Sun Sep 18 15:01:26 2011
@@ -212,11 +212,11 @@
                WorldServer var10 = this.field_419_b.func_28004_a(this.field_4110_as);
                this.field_422_ag.remove(var7);
                NetServerHandler var10000 = this.field_20908_a;
-               Packet51MapChunk var10001 = new Packet51MapChunk;
+               //Packet51MapChunk var10001 = new Packet51MapChunk;
                int var10003 = var7.field_152_a * 16;
                int var10005 = var7.field_151_b * 16;
                var10.getClass();
-               var10001.<init>(var10003, 0, var10005, 16, 128, 16, var10);
+               Packet51MapChunk var10001 = new Packet51MapChunk(var10003, 0, var10005, 16, 128, 16, var10);
                var10000.func_39_b(var10001);
                int var8 = var7.field_152_a * 16;
                var10003 = var7.field_151_b * 16;
diff -r -U 3 minecraft_server\net\minecraft\src\ItemMap.java minecraft_server_patched\net\minecraft\src\ItemMap.java
--- minecraft_server\net\minecraft\src\ItemMap.java	Sun Sep 18 14:38:34 2011
+++ minecraft_server_patched\net\minecraft\src\ItemMap.java	Sun Sep 18 15:01:54 2011
@@ -22,7 +22,7 @@
    }
 
    public MapData func_28023_a(ItemStack var1, World var2) {
-      "map_" + var1.func_21125_h();
+      //"map_" + var1.func_21125_h();
       MapData var4 = (MapData)var2.func_28103_a(MapData.class, "map_" + var1.func_21125_h());
       if(var4 == null) {
          var1.func_28145_b(var2.func_28104_b("map"));
diff -r -U 3 minecraft_server\net\minecraft\src\NetLoginHandler.java minecraft_server_patched\net\minecraft\src\NetLoginHandler.java
--- minecraft_server\net\minecraft\src\NetLoginHandler.java	Sun Sep 18 14:38:22 2011
+++ minecraft_server_patched\net\minecraft\src\NetLoginHandler.java	Sun Sep 18 15:01:24 2011
@@ -109,14 +109,14 @@
          ChunkCoordinates var4 = var3.func_22078_l();
          var2.field_425_ad.func_35695_b(var3.func_22081_n().func_35501_n());
          NetServerHandler var5 = new NetServerHandler(this.field_5_e, this.field_8_b, var2);
-         Packet1Login var10001 = new Packet1Login;
+         //Packet1Login var10001 = new Packet1Login;
          int var10004 = var2.field_331_c;
          long var10005 = var3.func_22079_j();
          int var10006 = var2.field_425_ad.func_35697_a();
          byte var10007 = (byte)var3.field_4272_q.field_6165_g;
          byte var10008 = (byte)var3.field_804_l;
          var3.getClass();
-         var10001.<init>("", var10004, var10005, var10006, var10007, var10008, (byte)-128, (byte)this.field_5_e.field_6033_f.func_35480_h());
+         Packet1Login var10001 = new Packet1Login("", var10004, var10005, var10006, var10007, var10008, (byte)-128, (byte)this.field_5_e.field_6033_f.func_35480_h());
          var5.func_39_b(var10001);
          var5.func_39_b(new Packet6SpawnPosition(var4.field_22216_a, var4.field_529_a, var4.field_528_b));
          this.field_5_e.field_6033_f.func_28170_a(var2, var3);
diff -r -U 3 minecraft_server\net\minecraft\src\ServerConfigurationManager.java minecraft_server_patched\net\minecraft\src\ServerConfigurationManager.java
--- minecraft_server\net\minecraft\src\ServerConfigurationManager.java	Sun Sep 18 14:38:24 2011
+++ minecraft_server_patched\net\minecraft\src\ServerConfigurationManager.java	Sun Sep 18 15:01:23 2011
@@ -188,12 +188,12 @@
       }
 
       NetServerHandler var10000 = var4.field_20908_a;
-      Packet9Respawn var10001 = new Packet9Respawn;
+      //Packet9Respawn var10001 = new Packet9Respawn;
       byte var10003 = (byte)var4.field_4110_as;
       byte var10004 = (byte)var4.field_9093_l.field_804_l;
       long var10005 = var4.field_9093_l.func_22079_j();
       var4.field_9093_l.getClass();
-      var10001.<init>(var10003, var10004, var10005, 128, var4.field_425_ad.func_35697_a());
+      Packet9Respawn var10001 = new Packet9Respawn(var10003, var10004, var10005, 128, var4.field_425_ad.func_35697_a());
       var10000.func_39_b(var10001);
       var4.field_20908_a.func_41_a(var4.field_322_l, var4.field_321_m, var4.field_320_n, var4.field_316_r, var4.field_315_s);
       this.func_28170_a(var4, var5);
@@ -218,12 +218,12 @@
       var1.field_4110_as = var11;
       WorldServer var4 = this.field_923_c.func_28004_a(var1.field_4110_as);
       NetServerHandler var10000 = var1.field_20908_a;
-      Packet9Respawn var10001 = new Packet9Respawn;
+      //Packet9Respawn var10001 = new Packet9Respawn;
       byte var10003 = (byte)var1.field_4110_as;
       byte var10004 = (byte)var1.field_9093_l.field_804_l;
       long var10005 = var4.func_22079_j();
       var4.getClass();
-      var10001.<init>(var10003, var10004, var10005, 128, var1.field_425_ad.func_35697_a());
+      Packet9Respawn var10001 = new Packet9Respawn(var10003, var10004, var10005, 128, var1.field_425_ad.func_35697_a());
       var10000.func_39_b(var10001);
       var2.func_22073_e(var1);
       var1.field_304_B = false;
